bin/github-backup

```diff
@@ -868,6 +875,26 @@ def backup_account(args, output_directory):
                      output_file,
                      account_cwd)
 
+    if args.include_followers or args.include_everything:
+        output_file = '{0}/followers.json'.format(account_cwd)
+        template = "https://{0}/users/{1}/followers"
+        template = template.format(get_github_api_host(args), args.user)
+        _backup_data(args,
+                    'followers',
+                    template,
+                    output_file,
+                    account_cwd)
+
+    if args.include_following or args.include_everything:
```

Comment:
Maybe consider extracting this download logic into a function rather than writing it twice